TestBenches/Macros.h

```diff
@@ -1176,7 +1176,55 @@ X(TP_L3L4D_, "TP_L3L4D") \
 X(TP_L5L6A_, "TP_L5L6A") \
 X(TP_L5L6B_, "TP_L5L6B") \
 X(TP_L5L6C_, "TP_L5L6C") \
-X(TP_L5L6D_, "TP_L5L6D") 
+X(TP_L5L6D_, "TP_L5L6D") \
```

Comment:
FWIW I think the xmacros could be simplified to just be X(TP_L5L6D) and then generate the underscore and the text substitution directly. This is more universally true for all X-macros. @bryates @aperloff